opm/parser/eclipse/EclipseState/Runspec.hpp

```diff
@@ -112,6 +112,69 @@ class WellSegmentDims {
     int nLatBranchMax;
 };
 
+class EclHysterConfig
+{
+public:
+      EclHysterConfig();
+      explicit EclHysterConfig(const Deck& deck);
+
+
+    /*!
+     * \brief Specify whether hysteresis is enabled or not.
+     */
+    void setEnableHysteresis(bool yesno);
```

Comment:
A general comment about `setXXX` methods in this class. I assume that the usage pattern for this class will be that it is not modified after the constructor has completed? In that case I would prefer if the assignments were done inline in the constructor, and the class as a whole had no public way to change state. Immutable is good!